src/components/recordset/recordset-table.tsx

```diff
@@ -29,12 +29,14 @@ import { addTopHorizontalScroll, fireCustomEvent } from '@isrd-isi-edu/chaise/sr
 type RecordsetTableProps = {
   config: RecordsetConfig,
   initialSortObject: any,
+  intersectScroll?: boolean,
```

Comment:
I'm assuming you used `intersectScroll` because you're using the `Intersection Observer`, but you should name it based on what it does not how it's implemented. Maybe something like `showOnlyOneHorizontalScrollbar` (which is inverse of what you have)? Or if you could come up with a better and shorter name?


Also please add a jsdoc style of comments to all the `type`s that you added the new prop:

```tsx
type Whatever = {
  /**
   * Explaing what's its purpose 
   * and if there's a default behavior also explain that here
   **/
  someProp?: Type
}
```
(I know that we should've added jsdoc to other props in these `type`s. But don't worry about existing props. Just add jsdoc for your own new prop).